src/Router.jsx

```diff
@@ -24,18 +24,17 @@ const Router = () => {
         { path: 'register', element: <Register /> },
         { path: '404', element: <NotFound /> },
         { path: '500', element: <Forbidden /> },
+        {
+          path: 'teams',
+          element: <ListTeams />,
+        },
       ],
     },
     {
-      path: '/team',
-      element: (
-        <RequireAuth>
-          <ConnectedLayout />
-        </RequireAuth>
-      ),
-      children: [{ path: '', element: <Home /> }],
+      path: '/teams',
+      element: <DefaultLayout />,
+      children: [{ path: 'user/:teamID', element: <AddUserToTeam /> }],
```

Comment:
Link with previous suggestion
```suggestion
      path: '/teams',
      element: (
        <RequireAuth>
          <ConnectedLayout/>
        </RequireAuth>
      ),
      children: [{ path: '', element: <ListTeams /> }, { path: '/:teamID', element: <AddUserToTeam /> }],
```